src/main/java/network/brightspots/rcv/RawContestConfig.java

```diff
@@ -80,10 +80,13 @@ public static class CvrSource {
     private SimpleStringProperty overvoteDelimiter = new SimpleStringProperty();
     private SimpleStringProperty provider = new SimpleStringProperty();
     private SimpleStringProperty overvoteLabel = new SimpleStringProperty();
-    private SimpleStringProperty undervoteLabel = new SimpleStringProperty();
+    private SimpleStringProperty skippedRankLabel = new SimpleStringProperty();
     private SimpleStringProperty undeclaredWriteInLabel = new SimpleStringProperty();
     private SimpleBooleanProperty treatBlankAsUndeclaredWriteIn = new SimpleBooleanProperty();
 
+    // Deprecated fields
```

Comment:
Oh, wow. Is this our first one ever? Or I guess we just didn't bother with logic like this before? E.g. I think `overvoteLabel` used to be config-level and then we changed it to source-level.

Reply:
First one in this class, but at the bottom of this file, you'll notice `ContestRules` has the same pattern